### PR TITLE
New UI announcement post

### PR DESCRIPTION
**Pull request by rebeccadee**
Blog post for new UI and visual design

https://app.asana.com/0/1199959077348208/1200175961766749/f

### CODE REVIEW

File: blogposts/2021/introducing-sourcegraphs-new-ui.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+title: Meet the new Sourcegraph UI
+description: "We're launching an entirely new visual design for the Sourcegraph UI today! We've added so many features to Sourcegraph in the last few years, that we needed an entirely new visual design to keep up with our advancements. Let's take a look at some of the new designs and how they will help users navigate, understand, and make changes to code more efficiently."
+author: Rob Rhyne
+authorUrl: /handbook/company/team#rob-rhyne
+publishDate: 2021-03-24T10:00-07:00
```
Comment (rebeccadee) [inline blogposts/2021/introducing-sourcegraphs-new-ui.md:6]:
```suggestion
publishDate: 2021-06-09T10:00-07:00
```

File: blogposts/2021/introducing-sourcegraphs-new-ui.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+![focus on code screenshot](https://sourcegraphstatic.com/blog/redesign/code.png)
+
+- Readability of code is paramount. To facilitate that, we’ve reduced non-essential UI elements to provide greater focus on the code itself.
+- Complicated interfaces can cause fatigue, so we've reduced UI elements overall and positioned the most important items so they’re unobtrusive and live exactly where they are needed.
+
+## Improved layouts
+
+![improved diff view screenshot](https://sourcegraphstatic.com/blog/redesign/layouts.png)
+
+- Layout factors heavily in the usability of a design. We've improved pages like diff diff views to make them easier to use and to help users quickly find the information they need.
```
Comment (content-hacker) [inline blogposts/2021/introducing-sourcegraphs-new-ui.md:44]:
```suggestion
- Layout factors heavily in the usability of a design. We've improved pages like diff views to make them easier to use and to help users quickly find the information they need. 
```

@rrhyne can you confirm this is a typo and only 1 `diff` is needed?

File: blogposts/2021/introducing-sourcegraphs-new-ui.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+## Consistent headers and navigation
+
+![new campaign headers screenshot](https://sourcegraphstatic.com/blog/redesign/headers.png)
+
+- Sourcegraph is rapidly becoming more than code search. A new global navigation provides immediate discoverability and access to current and future functionality.
+- Consistency improves usability. We've standardized headers to orient users and provide all available options in a single location.
+
+## Promoting extensibility
+
+![exntensions redesign screenshot](https://sourcegraphstatic.com/blog/redesign/exntensions.png)
```
Comment (content-hacker) [inline blogposts/2021/introducing-sourcegraphs-new-ui.md:56]: @rrhyne this image isn't rendering in the preview for me. Are you having the same issue?

Comment (content-hacker) [inline blogposts/2021/introducing-sourcegraphs-new-ui.md:56]:
```suggestion
![exntensions redesign screenshot](https://sourcegraphstatic.com/blog/redesign/extensions.png)
```

Comment (content-hacker) [reply blogposts/2021/introducing-sourcegraphs-new-ui.md]: I noticed `extensions` was spelled incorrectly in the file path. I just updated it to see if that fixes the issue.

**Pull request merged**